src/main/java/com/frcteam3255/joystick/SN_F310Gamepad.java

```diff
@@ -149,11 +151,14 @@ public double getAxisRSY() {
 	}
 	
 	/**
-	 * @param axisId id number of controller axis, viewable in this file if needed
-	 * @return position value of RawAxis(axisId)
+	 * @param axisId SN_IntPreference
+	 * @return position value of axis
 	 */
-	public double getAxisVar(int axisId) {
-		return getRawAxis(axisId);
+	public double getAxisVar(SN_IntPreference axisId) {
+
+		int intAxisId = axisId.getValue();
+
+		return getRawAxis(intAxisId);
```

Comment:
Save yourself a step and a bit of memory
```suggestion
		return getRawAxis(axisId.getValue());
```